stable/grafana/templates/poddisruptionbudget.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.podDisruptionBudget.minAvailble }}
```

Comment:
Hmm I think it might be better to condition this on a new key, `.Values.podDisruptionBudget.enabled`. That way you can later introduce a `.Values.podDisruptionBudget.maxUnavailable` too

Reply:
I was thinking on a rewrite if we want to add maxUnavailable as well. I didn't want to have an enable/disable flag though. I'll try to figure out another method